Sources/StreamVideoSwiftUI/CallViewModel.swift

```diff
@@ -77,12 +77,20 @@ open class CallViewModel: ObservableObject {
     public var error: Error? {
         didSet {
             errorAlertShown = error != nil
+            if let error {
+                toast = Toast(style: .error, message: error.localizedDescription)
+            } else {
+                toast = nil
+            }
         }
     }
+        
+    /// Controls the display of toast messages.
+    @Published public var toast: Toast?
     
     /// If the `error` property has a value, it's true. You can use it to control the visibility of an alert presented to the user.
     @Published public var errorAlertShown = false
```

Comment:
Do we still need this one?

Reply:
Yes, for 2 reasons:
- in case someone wants to show classic alert
- `Toast` is iOS 14 and above, so it's still used in iOS 13